### DISCUSSION TITLE
There are many abbreviations in the code, not easy for newbies to understand the code.

### DISCUSSION THREAD

**Discussion by fuchencong** [loxilb-io/loxilb#712] [General]
Hi folks,

Thank all of you for hard work in this great project. Recently I have been learning our project code, and also wrote an [Chinese blog ]( https://fuchencong.com/2024/06/27/loxilb-01/ )about  loxilb project. 

As a newbie, I met some trouble to understand the project code because there are  many abbreviations in code. For example, it's difficult to guess the meaning of field "has_pb" in struct l "lb_dp_map".  Is there any way to help me understand the code abbreviation quickly?

Hope developers can add some code comments, especially for uncommon abbreviation , which may help more developers to join the project.

Thanks.

**Comment by UltraInstinct14** [comment]
Thanks @fuchencong  for writing a wonderful blog. We try our best to maintain the best possible code [comments/docs](https://pkg.go.dev/github.com/loxilb-io/loxilb) as per Go standards.  For your specific question - 

"has_pb" indicates if an eBPF map has some associated per-CPU packet and(or) byte counter map. As you could probably understand expanding it would make the names simply too long.  

We did go through documentation/readability improvement [activity](https://github.com/loxilb-io/loxilb/issues/61), but still, there is room for improvement. If there is something you would like to change for better readability, feel free to change and send a pull-request. Will be happy to review and merge !!
